tests/Integration/Integration.Tests/VendorManagersModule/ContextBase.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace Linn.Purchasing.Integration.Tests.VendorManagersModule
+{
+    using System.Net.Http;
+
```

Comment:
I think Stylecop should want usings ordered alphabetically too. I don't really care but maybe your Stylecop has stopped warning you?

Reply:
ignore, I forgot it wants System. ones first